### PR TITLE
Create cs operator subscription when cs operator doesn't exist in nam…

### PR DESCRIPTION
**Pull request by DanielXLee**
…espace ibm-common-service

### CODE REVIEW

File: bundle/manifests/ibm-common-service-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -489,36 +503,56 @@ metadata:
...
-                value: "secretshare"
-              resources:
-                limits:
-                  cpu: 500m
-                  memory: 512Mi
-                requests:
-                  cpu: 200m
-                  memory: 200Mi
-            terminationGracePeriodSeconds: 10
+              - name: ansible
```
Comment (horis233) [inline bundle/manifests/ibm-common-service-operator.clusterserviceversion.yaml:507]: @DanielXLee Do we need to change the secretshare and cs webhook in the CSV?

File: config/manifests/bases/ibm-common-service-operator.clusterserviceversion.yaml [hunk 2]
```diff
@@ -467,7 +481,7 @@ metadata:
               productID: 068a62892a1e4db39641342e592daa25
               productMetric: FREE
               productName: IBM Cloud Platform Common Services
-              productVersion: 3.4.0
+              productVersion: 3.5.0
```
Comment (horis233) [inline config/manifests/bases/ibm-common-service-operator.clusterserviceversion.yaml:484]: @DanielXLee ProductVersion would be removed in 3Q.

File: bundle/manifests/ibm-common-service-operator.clusterserviceversion.yaml (hunk 1 above)
Comment (DanielXLee) [reply bundle/manifests/ibm-common-service-operator.clusterserviceversion.yaml:507]: I don't change any code for secretshare and webhook, This bundle manifests generated by template https://github.com/IBM/ibm-common-service-operator/blob/master/config/manifests/bases/ibm-common-service-operator.clusterserviceversion.yaml, maybe I need to update this template.

File: config/manifests/bases/ibm-common-service-operator.clusterserviceversion.yaml (hunk 2 above)
Comment (DanielXLee) [reply config/manifests/bases/ibm-common-service-operator.clusterserviceversion.yaml:484]: okay

File: controllers/bootstrap/init.go [hunk 3]
```diff
@@ -77,6 +78,30 @@ func InitResources(mgr manager.Manager) error {
 		return err
 	}
 
+	// Checking existing CommonService subscription
```
Comment (chenzhiwei) [inline controllers/bootstrap/init.go:81]: if ns is not in openshift-operators or ibm-common-services {
  create cs subscription
  return
}

**Pull request merged**